hadoop-hdds/docs/content/interface/OzoneFS.md

```diff
@@ -36,7 +36,7 @@ ozone sh volume create /volume
 ozone sh bucket create /volume/bucket
 {{< /highlight >}}
 
-Once this is created, please make sure that bucket exists via the listVolume or listBucket commands.
+Once this is created, please make sure that bucket exists via the _list volume_ or _list bucket_ commands.
```

Comment:
The corresponding `ozone sh` sub-commands are actually `volume list` and `bucket list`.

Reply:
Thanks, will fix it.